### PR TITLE
NSKOPS-308: Changed settings for es-1.7.5

### PR DESCRIPTION
**Pull request by ghostik**
In this PR was changed some settings for a compatibility the package with es-1.7.x versions.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
     <groupId>org.elasticsearch</groupId>
     <artifactId>elasticsearch-srv-discovery</artifactId>
-    <version>1.7.1-1</version>
+    <version>1.7.1-2</version>
```
Comment (mkostrikin) [inline pom.xml]: Why not `1.7.5-1` ?

File: pom.xml [hunk 2]
```diff
@@ -25,7 +25,7 @@
     </scm>
 
     <properties>
-        <elasticsearch.version>1.7.1</elasticsearch.version>
+        <elasticsearch.version>1.7.5</elasticsearch.version>
```
Comment (mkostrikin) [inline pom.xml]: What that version is for ?

File: src/deb/control/control [hunk 3]
```diff
@@ -11,4 +11,4 @@ Architecture: all
 Conflicts:
 Replaces:
 Description: [[description]]
-Depends: elasticsearch (= 1.7.1)
+Depends: elasticsearch (>= 1.7.1)
```
Comment (mkostrikin) [inline src/deb/control/control]: `>= 1.7.0` ?

**Comment by mkostrikin** [comment]
Please, either keep compatibility with `1.7.x` or make version which required only particular ES version.

**Comment by mkostrikin** [comment]
If do package with stroing dependency on 1.7.5, then we have to create a corresponding branch and re-do PR.

File: src/deb/control/control (hunk 3 above)
Comment (mkostrikin) [inline src/deb/control/control]: Also, requiremend that version must be `< 2.x`

Comment (ghostik) [inline src/deb/control/control]: changed https://github.com/Tradeshift/elasticsearch-srv-discovery/pull/9/commits/5755b41d3ea09ea59f20da2734b2307402dd7eb2#diff-600376dffeb79835ede4a0b285078036R35

Comment (ghostik) [inline src/deb/control/control]: and https://github.com/Tradeshift/elasticsearch-srv-discovery/pull/9/files#diff-fb6b04e686c1b6e22755d4c7ececf443R14

File: pom.xml [hunk 4]
```diff
@@ -32,7 +32,7 @@
         <dependency>
             <groupId>org.elasticsearch</groupId>
             <artifactId>elasticsearch</artifactId>
-            <version>${elasticsearch.version}</version>
+            <version>[${elasticsearch.version},2.0.0-!)</version>
```
Comment (ashilokhvostov) [inline pom.xml]: ! or 1 ?

Comment (ghostik) [inline pom.xml]: -!

File: pom.xml (hunk 1 above)
Comment (ghostik) [inline pom.xml]: 1.7.5-1 already exist.
```
root@logstash-search-i-vagranto:/home/ktu# apt-cache policy elasticsearch-srv-discovery
elasticsearch-srv-discovery:
  Installed: (none)
  Candidate: 1.7.1+1
  Version table:
     1.7.1+1 0
        500 http://pkg.tradeshift.com/ubuntu/ trusty/main amd64 Packages
     1.4.2+1 0
        500 http://pkg.tradeshift.com/ubuntu/ trusty/main amd64 Packages
     0.90.11+8 0
        500 http://pkg.tradeshift.com/ubuntu/ trusty/main amd64 Packages
     0.90.11+7 0
        500 http://pkg.tradeshift.com/ubuntu/ trusty/main amd64 Packages
     0.90.11+6 0
        500 http://pkg.tradeshift.com/ubuntu/ trusty/main amd64 Packages
     0.90.11+5 0
        500 http://pkg.tradeshift.com/ubuntu/ trusty/main amd64 Packages
     0.90.11+4 0
        500 http://pkg.tradeshift.com/ubuntu/ trusty/main amd64 Packages
     0.90.11+3 0
        500 http://pkg.tradeshift.com/ubuntu/ trusty/main amd64 Packages
     0.90.11+2 0
        500 http://pkg.tradeshift.com/ubuntu/ trusty/main amd64 Packages
     0.90.11+1 0
        500 http://pkg.tradeshift.com/ubuntu/ trusty/main amd64 Packages
```

File: pom.xml (hunk 2 above)
Comment (ghostik) [inline pom.xml]: https://github.com/Tradeshift/elasticsearch-srv-discovery/pull/9/files#diff-600376dffeb79835ede4a0b285078036R35

File: src/deb/control/control (hunk 3 above)
Comment (ghostik) [inline src/deb/control/control]: In ts repo ES version 1.7.0 is absent.
```
root@logstash-search-i-vagranto:/home/ktu# apt-cache policy elasticsearch
elasticsearch:
  Installed: 2.2.1
  Candidate: 2.2.1
  Version table:
 *** 2.2.1 0
        500 http://pkg.tradeshift.com/ubuntu/ trusty/main amd64 Packages
        100 /var/lib/dpkg/status
     2.2.0 0
        500 http://pkg.tradeshift.com/ubuntu/ trusty/main amd64 Packages
     2.1.1 0
        500 http://pkg.tradeshift.com/ubuntu/ trusty/main amd64 Packages
     1.7.5 0
        500 http://pkg.tradeshift.com/ubuntu/ trusty/main amd64 Packages
     1.7.1 0
        500 http://pkg.tradeshift.com/ubuntu/ trusty/main amd64 Packages
     1.4.4 0
        500 http://pkg.tradeshift.com/ubuntu/ trusty/main amd64 Packages
     1.4.2 0
        500 http://pkg.tradeshift.com/ubuntu/ trusty/main amd64 Packages
     0.90.11 0
```

**Comment by ghostik** [comment]
An answer on questions in
https://github.com/Tradeshift/elasticsearch-srv-discovery/pull/9#issuecomment-208338480
https://github.com/Tradeshift/elasticsearch-srv-discovery/pull/9#issuecomment-208338841
are in 
https://github.com/Tradeshift/elasticsearch-srv-discovery/pull/9/commits/5755b41d3ea09ea59f20da2734b2307402dd7eb2
https://github.com/Tradeshift/elasticsearch-srv-discovery/pull/9/commits/1d478ceae4d03a044fe02765403db5f6111b8057

File: pom.xml (hunk 1 above)
Comment (mkostrikin) [inline pom.xml]: I do not see `1.7.5-1`

Comment (ghostik) [inline pom.xml]: My mistake. The version 1.7.5-1 is absent. In this PR an interval of ES versions was extended (1.7.1<=version<2.0.0 ). I think, it is logical to name this version 1.7.1-x. The version 1.7.1-1 is present already. It is reasons why this package was named 1.7.1-2.

Comment (mkostrikin) [inline pom.xml]: good

**Comment by mkostrikin** [comment]
:floppy_disk: 
squash and ask for external review

File: pom.xml (hunk 4 above)
Comment (diogokiss) [inline pom.xml]: Does it actually work?

**Comment by rodrigodealer** [comment]
💯 You're good to go.

**Pull request merged**